go/tools/semconvgen/README.md

```diff
@@ -0,0 +1,23 @@
+# Go Semantic Convention Generator
+
+This tool is designed to generate constants in a semantic convention package for the Go API and the collector.
+It may be used by other systems, but it's primary function beyond invoking the template processor is to ensure that
+generated identifiers conform to Go's naming idiom, particularly with respect to initialisms and acronyms.
+Other users may be served just as well by using the template processor directly.
+
+## Usage
+
+```shell
+$ semconvgen -i <path to spec YAML> -t <path to template> -o <path to output>
```

Comment:
The Python tool is invoked with `gen-semconv`, so maybe use a name with the same scheme, like `go-gen-semconv`

Reply:
+1 to renaming this to go-gen-semconv.   We can follow similar conventions for other langaauge SiGs that want/need specialized codegen.